config.toml

```diff
@@ -0,0 +1,8 @@
+main_backend = "main"
```

Comment:
This file should be added to the `extra-source-files` section in `package.yaml`.

Reply:
Why was `config.toml` deleted?

Reply:
Done